src/SignalR/clients/ts/signalr-protocol-msgpack/src/MessagePackHubProtocol.ts

```diff
@@ -32,6 +32,16 @@ export class MessagePackHubProtocol implements IHubProtocol {
     private readonly voidResult = 2;
     private readonly nonVoidResult = 3;
 
+    /**
+     *
+     * @param messagePackOptions MessagePack options passed to msgpack5
+     */
+    constructor(public messagePackOptions?: any) {
```

Comment:
This should be `private messagePackOptions?: any`

Reply:
Thanks! I have updated this to private.